cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-db-operations.asciidoc

```diff
@@ -0,0 +1,44 @@
+The rules of a security model can have some impact on some of the database operations.
+This comes down to necessary additional security checks that incur additional data accesses.
+Especially when it comes to count store operations, as they are usually very fast lookups, this difference might be noticeable.
+
+Lets look at the following security rules that setup a "restricted" and a "free" role as an example:
```

Comment:
Maybe the role names "restricted" and "free" should be backticked instead of in quotes? Same in the table header below